static/scripts/helpers/AlertMessageController.js

```diff
@@ -55,7 +55,12 @@ class AlertMessageController {
 
 	showAlert(messageArray) {
 		if (!messageArray || messageArray.length === 0) {
-			return; // If messageArray is empty, we do not show the triangle
+			// If messageArray is empty, we do not show the triangle
+			localStorage.setItem('SC-Alerts', JSON.stringify([]));
+			if (this.loggedin) {
```

Comment:
Der if block ist etwas hacky, da es sich darauf verlässt das keine weitere Änderung eintritt, bis auf eine neue Seite geklickt wird. Aber für das server side rendering denke ich ist es ok.

Reply:
Hatte auch erst überlegt, dass Handling "vor" die If-Schleife zu ziehen. Das würde aber bedeuten, dass wir die "Alerts" immer erst leeren und dann neusetzen, auch wenn sich eigentlich nichts ändern. Wollte das entstehende "Flackern" verhindern und habe daher die Sonderbehandlung in das If mit reingezogen.